service/src/test/java/bio/terra/workspace/connected/LandingZoneTestUtils.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.workspace.connected;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("connected-test")
+public class LandingZoneTestUtils {
+  @Value("${workspace.connected-test.default-landing-zone-id}")
```

Comment:
I think this should be in `azure-connected-test` and not `connected-test`

In general, we avoid `@Value`, but this seems like a reasonable usage.